src/WhoReacted/index.jsx

```diff
@@ -58,6 +62,18 @@ export default class WhoReacted extends Plugin {
                     this.settings.maxUsersShown = parseInt(value);
                 }
             ),
+            new Slider(
```

Comment:
We probably should start creating categories to organize these settings.

Reply:
Are those the collabsable sections seen in other plugins' configurations? So far, I think we can configure into "when" and "what".

Reply:
Yeah, we could divide them into sections of `Display Settings`, `Filter Settings`, and `Threshold Settings`.

Reply:
I'll wait for the merges in the zere's library to start organizing these settings.